foundation/org.eclipse.persistence.core/src/org/eclipse/persistence/config/PersistenceUnitProperties.java

```diff
@@ -2599,6 +2599,14 @@
      */
     public static final String ALLOW_NULL_MAX_MIN = "eclipselink.allow-null-max-min";
 
+    /**
+     * The "<code>eclipselink.sql.allow-convert-result-to-boolean</code>" property allows
+     * eclipselink to convert resultset values to boolean types.
+     *
+     * Default: "<code>false</code>".
```

Comment:
If the option is being added for backward compatibility (and I believe that is the case here) then the default should be `true` instead